daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/dfa/Rules.scala

```diff
@@ -501,6 +503,11 @@ class EECStateBlock(states: => ArrayBuffer[State], blockEnd: DFADelimiter, val E
         r.nextState = StartState
       }
     },
+    Rule { (r: Registers) => couldBeFirstChar(r.data0, blockEnd) } {
+      (r: Registers) => {
+        r.status = StateKind.Paused //PTERMState
+      }
+    },
```

Comment:
Can you clarify why this change is needed. I can't say if it's write or wrong, but seems odd that this changes it to check data1 first before data0. Does this instead need a change similar to the above?

Reply:
Can we add a test that we know drives these rules, so we can see if this data1 first thing is a bug? (I think it is.)